csm_web/scheduler/views/section.py

```diff
@@ -638,6 +646,31 @@ def _student_add(self, request, section):
                 log_str(section),
                 log_str(old_section),
             )
+
+            # Send swap email
+            try:
+                email_swap(student)
+                logger.info(
+                    f"<Enrollment Email:Success> Email for {student} swapping sent"
+                )
+            except NoEmailError:
+                mentor = student.section.mentor
+                logger.info(
+                    f"<Enrollment Email:Failure> Email address for {mentor} not found"
+                )
+            except EmailFormattingError:
+                logger.info(
+                    f"<Enrollment Email:Failure> Email has not been formatted correctly for sending"
+                )
+            except EmailAuthError:
+                logger.info(
+                    f"<Enrollment Email:Failure> Cannot log into CSM email"
+                )
+            except HttpError:
+                logger.info(
+                    f"<Enrollment Email:Failure> Email failed to send"
+                )
+
```

Comment:
This block of code seems to be copied a lot throughout the views; perhaps look into moving this into a separate function that handles all of the errors? The main concern is the amount of duplicate code here, and to allow for future additions/modifications across all instances of error handling, etc.